bundles/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/local/TelldusLocalDeviceController.java

```diff
@@ -270,7 +272,8 @@ private <T> T innerCallRest(String uri, Class<T> json)
             throws ExecutionException, InterruptedException, TimeoutException, JsonSyntaxException {
         logger.trace("HTTP GET: {}", uri);
 
-        Request request = httpClient.newRequest(uri).method(HttpMethod.GET);
+        Request request = httpClient.newRequest(uri).method(HttpMethod.GET).timeout(REQUEST_TIMEOUT_MS,
+                TimeUnit.MICROSECONDS);
```

Comment:
```suggestion
        Request request = httpClient.newRequest(uri).method(HttpMethod.GET).timeout(REQUEST_TIMEOUT_MS,
                TimeUnit.MILLISECONDS);
```

Reply:
Thanks, fixed

Reply:
Not yet fixed.

Reply:
Weird, don't know what happened, anyway fixed.